recipes/param_persist/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "param_persist" %}
+{% set version = "0.1.1" %}
+{% set sha256 = "fcf7180f544236683fb96b8fc80c752b8d34ec153abd4b54024c28cff1670bd5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ os_distribution }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python >=3.7
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - sqlalchemy
```

Comment:
For 99% of Python packages, `host` and `run` should have the same `python` requirement:
```suggestion
    - python >=3.6
    - sqlalchemy
```

Reply:
I agree to be a maintainer